src/main/java/edu/ie3/datamodel/io/source/couchbase/CouchbaseWeatherSource.java

```diff
@@ -216,6 +247,19 @@ public String createQueryStringForIntervalAndCoordinate(
     return basicQuery + whereClause;
   }
 
+  /**
+   * Create a query string to search for all time keys that comes after the given time.
+   *
+   * @param time given timestamp
+   * @return the query string
+   */
+  public String createQueryStringForFollowingTimeKeys(ZonedDateTime time) {
+    String basicQuery =
+        "SELECT " + connector.getBucketName() + ".* FROM " + connector.getBucketName();
+    String whereClause = " WHERE META().id > '" + generateWeatherKey(time, 0) + "'";
```

Comment:
Why coordinateId within generateWeatherKey is fixed to 0?

Reply:
This will check all coordinateIds >= 0.